app/environments/brassbirmingham/brassbirmingham/envs/classes/player.py

```diff
@@ -90,18 +104,21 @@ def decreaseLevel():
         for _ in range(levels):
             decreaseLevel()
 
+    # pass "money" object
     def canAffordBuildingIndustryResources(
-        self, buildLocation: BuildLocation, coalCost: int, ironCost: int
+        self, buildLocation: BuildLocation, coalCost: int, ironCost: int, money: int
```

Comment:
If this is a non-static function for Player, why do we need to pass `money` in? I think relying on `self.money` made more sense?

Reply:
In this case it makes sense to pass it in, it'd be the remaining money (to spend on coal/iron) after spending on the building's cost, I'll rename it to `moneyRemaining` or something

Reply:
Hmm, I suppose that makes sense. What about keeping it non-static but take in a variable called ‘additionalSpentMoney’ or ‘additionalCost’ or something. Then subtract that from self money for the check

Reply:
I changed it to accept `building` as the 2nd parameter, and to pull all the costs from there